Logger.md

```diff
@@ -5,13 +5,13 @@
 ## Declaration
 
 ```swift
-public enum LogLevel {
+@objc public enum LogLevel: Int {
     case info
     case warning
     case error
 }
 
-public protocol Logger {
+@objc public protocol Logger {
```

Comment:
I think we should be adding a prefix when translating things to Obj-C --> `@objc(MJLogger), and this should apply to all @objc decorators.

Reply:
I've used **HM**Logger, instead of **MJ**. I've updated the reference repository too.

Reply:
Requested changes implemented